FilterJointTrajectoryWithConstraintsRequest filter_request
---
trajectory_msgs/JointTrajectory trajectory
ArmNavigationErrorCodes error_code